src/category_theory/limits/shapes/equalizers.lean

```diff
@@ -210,6 +210,12 @@ abbreviation fork.ι (t : fork f g) := t.π.app zero
     `t.ι.app zero : X ⟶ t.X` and `t.ι.app one : Y ⟶ t.X`. Of these, only the second one is
     interesting, and we give it the shorter name `cofork.π t`. -/
 abbreviation cofork.π (t : cofork f g) := t.ι.app one
+
+@[simp] lemma fork.ι_of_ι {P : C} (ι : P ⟶ X) (w : ι ≫ f = ι ≫ g) :
+  fork.ι (fork.of_ι ι w) = ι := rfl
```

Comment:
Would it work to mark `fork.of_ι` (and `cofork.of_π`) as `@[simps]` so that these two don't need to be here?

Reply:
`fork.ι` isn't a structure projection (it is defined as `t.π.app zero`, where `t` is a fork), so `@[simps]` does not work here.